src/types/PreparedPost.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from './User';
+import { Comment } from './Comment';
+
+export interface PostAllInfo {
```

Comment:
if you create PreparedPost file name, will be better to name the type PreparedPost as well